tests/gatekeeper/valid/policy.yaml

```diff
@@ -32,7 +32,6 @@ kind: K8sBannedConfigMapKeysV1
 metadata:
   name: no-secrets-in-configmap
 spec:
-  enforcementAction: dryrun
```

Comment:
why is htis removed here and added above?

Reply:
This is not true. GitHub diff appears to be not smart enough.
What happened is:
- tests/gatekeeper/valid/resources.yaml => tests/gatekeeper/valid/policy.yaml
- tests/gatekeeper/dryrun/resources.yaml => tests/gatekeeper/dryrun/policy.yaml

But GH diff thought it's:
- tests/gatekeeper/valid/resources.yaml => tests/gatekeeper/dryrun/policy.yaml
- tests/gatekeeper/dryrun/resources.yaml => tests/gatekeeper/valid/policy.yaml